validator/sawtooth_validator/journal/genesis.py

```diff
@@ -154,13 +162,20 @@ def start(self, on_done):
         LOGGER.debug('Produced state hash %s for genesis block.',
                      state_hash)
 
+        block_builder = GenesisController._generate_genesis_block()
         block_builder.add_batches(genesis_batches)
         block_builder.set_state_hash(state_hash)
 
+        block_publisher = self._get_block_publisher(state_hash)
+        block_publisher.initialize_block(block_builder)
```

Comment:
This should probably be the raw block or a BlockWrapper instead of the BlockBuilder.